src/isomedia/box_code_base.c

```diff
@@ -12450,6 +12450,7 @@ void ddts_box_del(GF_Box *s)
 GF_Err ddts_box_read(GF_Box *s, GF_BitStream *bs)
 {
 	GF_DTSSpecificBox *ptr = (GF_DTSSpecificBox *)s;
```

Comment:
I forgot that in the prev review: box parsing code should use the macro ISOM_DECREASE_SIZE when reading, so your code should just be (preferably before first gf_bs_read):
ISOM_DECREASE_SIZE(ptr, 20)

you then do not need to check if there are remaining bytes, this is done by the caller

Reply:
I uploaded a new commit with the suggested changes, and that looks a lot nicer.  Thanks!